lib/opal/nodes/def.rb

```diff
@@ -199,7 +199,10 @@ def compile
               scope.mlhs_mapping[child] = tmp
 
               child.children.each do |child|
-                scope.add_temp variable(child.last)
+                # No support for nested mlhs yet
+                if child.type != :mlhs
+                  scope.add_temp variable(child.last)
+                end
```

Comment:
This code raises an error on the compilation stage and makes `ruby/spec/language/block_spec` unparsable. Could we just ignore nested `mlhs` arguments for now?